src/orgwarden/__main__.py

```diff
@@ -98,6 +98,16 @@ def audit(
             show_default=False,
         ),
     ] = None,
+    modules_str: Annotated[
```

Comment:
Consider taking modules as an optional list of strings (e.g. `modules: Annotated[list[str] | None, ...]`). The command line would then be `--module GitHub --module GitHubCustomization`.

Reply:
Moving to a list of strings feels a bit cleaner to me, as a list of strings will be more consistent with other repeated arguments.